lib/railway_ipc/application.ex

```diff
@@ -2,11 +2,12 @@ defmodule RailwayIpc.Application do
   @moduledoc false
 
   use Application
-
+  @dev_repo Application.get_env(:railway_ipc, :dev_repo)
```

Comment:
So I understand that `config.exs` is not loaded when this app is included as a dependency of another application _but_ i don't get why config that I set at compile-time evaluates to nil when railway runs as a dependency of another app (in this case course conductor). That is why I had to do some janky logic here to ensure that the dev repo only gets set under the right circumstances. @StevenNunez